### PR TITLE
Cloud SDK: Tutorial to integrate with CAP Framework

### PR DESCRIPTION
**Pull request by MatKuhr**
This PR is a replacement for #5739 with the same content.

### CODE REVIEW

**Comment by sacnl** [comment]
@thecodester 
As this has been discussed in the previous pull request, I assume this is good to go?
As we already had users asking for this tutorial, I would like to merge this by tomorrow EOB, unless you see open issues.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
This tutorial has an unreachable link in line 48.
It also has outdated author information (Nabil shafi is no longer with SAP).
Please update the tutorial and validate it using the Atom plugin.

**Comment by MatKuhr** [comment]
> This tutorial has an unreachable link in line 48.

Please check again, for me this link is reachable. 

> It also has outdated author information (Nabil shafi is no longer with SAP).

How can I change the author information? I already changed the commits such that only I occur in the commit history.

> Please update the tutorial and validate it using the Atom plugin.

Validation is successful with my plugin.

File: tutorials/cloudsdk-integrate-cap/cloudsdk-integrate-cap.md [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+  - How to integrate the  SAP Cloud SDK into the SAP Cloud Application Programming Model
+  - How to write a custom event handler for CAP Java
+  - How to deploy an application to SAP Cloud Platform Cloud Foundry
+  - How to create service that reads/writes business partners from S/5HANA and exposes them as SAP Cloud Application Programming Model
+
+SAP Cloud Application Programming Model enables you to quickly create business applications by allowing you to focus on your business domain. It offers a consistent end-to-end programming model for full-stack development on SAP Cloud Platform.
+
+However, this raises the question how this interacts and integrates with other SAP frameworks that SAP has introduced over the past year such as the SAP Cloud SDK.
+
+In particular, you will learn how to integrate the SAP Cloud SDK into the SAP Cloud Application Programming Model. And how SAP Cloud SDK allow you to develop, extend and communicate with other SAP solutions.
```
Comment (MichaelCzcz) [inline tutorials/cloudsdk-integrate-cap/cloudsdk-integrate-cap.md:26]:
```suggestion
In particular, you will learn how to integrate the SAP Cloud SDK into the SAP Cloud Application Programming Model. And how SAP Cloud SDK allows you to develop, extend and communicate with other SAP solutions.
```

File: tutorials/cloudsdk-integrate-cap/cloudsdk-integrate-cap.md [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 6: ](Understanding the read and write methods)]
+
+The above class handles the READ and CREATE events (highlighted above).
+
+- The READ operation just returns the business partners from the remote OData service.
+
+    Here we are using the OData VDM from SAP Cloud SDK to fetch business partners from the target system. We can call this service `getAllBusinessPartner()` function to retrieve a list of all the business partners from the system. In this tutorial our destination is called `MyErpSystem` and you will learn a little later where it is set up.
```
Comment (MichaelCzcz) [inline tutorials/cloudsdk-integrate-cap/cloudsdk-integrate-cap.md:243]: Always say "you will" or "your system", never "we".

Please change all occurrences accordingly.
Please see the style guide for more details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9

File: tutorials/cloudsdk-integrate-cap/cloudsdk-integrate-cap.md [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+
+    Later we transform `BusinessPartner` to `CapBusinessPartner` according to our requirements.
```
Comment (MichaelCzcz) [inline tutorials/cloudsdk-integrate-cap/cloudsdk-integrate-cap.md:245]: Always say "you will" or "your system", never "we".

Please change all occurrences accordingly.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by MatKuhr** [comment]
I addressed the changes and also added my GH profile as author profile.

**Pull request merged**